scripts/run

```diff
@@ -38,4 +38,23 @@ if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
   sed -e 's/ / +/g' -e 's/^/+/' <"/sys/fs/cgroup/cgroup.controllers" >"/sys/fs/cgroup/cgroup.subtree_control"
 fi
 
+if [ $CI ]; then
+  # Attempt to find and stop any previous instances of dapper containers.
+  # To combat the scenario where drone runners are reused and an instance of rancher is still running.
+  echo "Checking for running containers from a previous build."
+
+  # gets a list of all docker containers that were started in a CI build.
+  dapper_containers=$(docker ps --format '{{.Names}}/{{.Label "DRONE_BUILD_NUMBER"}}' --filter 'label=CI=true')
+  for conatiner_info in $dapper_containers; do
```

Comment:
Also this looks like a typo, but it's used consistently below, so... it's not **broken** or anything.

Reply:
Going to leave it so that we don't have to run CI again 😄

Reply:
nmv fixed